src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -191,122 +155,141 @@ public String  MakeGUID() {
 
     public void BeginConnectPeer(String toPeerId, ConnectStatusCallback connectStatusCallback) {
```

Comment:
This one is really optional but when I'm dealing with multi-threaded code I will tend to put a final on my method arguments (e.g. final String toPeerId) just to make sure they are immutable and I don't accidentally mess with them in the code. But this is a matter of taste.